nova_plugin/server.py

```diff
@@ -292,10 +292,11 @@ def rename(name):
         # Make sure that if the server is connected to a security group
         # from CREATE time so that there the user can control
         # that there is never a time that a running server is not protected.
-        security_groups = \
-            get_openstack_ids_of_connected_nodes_by_relationship_type(
-                ctx, 'cloudify.openstack.server_connected_to_security_group')
-        server['security_groups'] = security_groups
+        security_group_names = \
+            get_attribute_of_connected_nodes_by_relationship_type(
```

Comment:
The OPENSTACK_SECURITY_GROUP_TYPE variable that is used elsewhere doesn't correspond to the expected value.